contents/tutorials/when-and-how-to-run-group-targeted-a-b-tests.md

```diff
@@ -0,0 +1,88 @@
+---
+title: When and how to run group-targeted A/B tests
+sidebar: Docs
+showTitle: true
+author: ['lior-neu-ner']
+date: 2023-04-21
+featuredImage: ../images/tutorials/when-and-how-to-run-group-targeted-a-b-tests/user-vs-group-a-b-tests.png
+tags: ['experimentation']
+---
+
+A/B tests are a powerful tool for measuring how product changes impact user behavior. However, sometimes changing how one user interacts with your product will affect how others use it too. For example, a change in the Uber App for Drivers will likely affect riders' experience too. In these scenarios, group-targeted experiments enable us to measure the impact of changes beyond an individual user and across a group of users.
+
+## What are group-targeted experiments
+
+In group-targeted experiments, the changes being tested are applied to an entire group, rather than individual users. This means that everyone within a specific group will experience the same conditions. For example, either the control or test variant. This enables you to measure the impact of any changes on the group as a whole, rather than on individual users within the group.
```

Comment:
```suggestion
In group-targeted experiments, the test applies to an entire group, rather than individual users. This means that everyone within a specific group will experience the same conditions, for example, either the control or test variant. This enables you to measure the impact of any changes on the group as a whole, rather than on individual users within the group.
```
1. Make sure to use [active voice](https://writing.wisc.edu/handbook/style/ccs_activevoice/)
2. "This enables you to measure the impact of any changes on the group as a whole, rather than on individual users within the group." is very similar to "In these scenarios, group-targeted experiments enable us to measure the impact of changes beyond an individual user and across a group of users." from the previous. You could probably cut or change one of them.